content/chainguard/chainguard-images/getting-started/getting-started-python.md

```diff
@@ -148,7 +148,9 @@ You have successfully completed the single-stage Python Chainguard Image. At thi
 
 ## Example 2 — Multi-Stage Build for Python Chainguard Image
 
-In this example, we'll build and run a multi-stage Python Chainguard Image to demonstrate the process that includes a build image with pip and a shell, and a final distroless image.
+In this example, we'll build and run a multi-stage Python Chainguard Image. We'll have a build image
+that includes pip and a shell before creating a final distroless image without these development
```

Comment:
sometimes paragraphs get split onto multiple lines like this, but it doesn't seem to matter on the front end? If I'm being a stickler I'd say remove the line breaks but nbd.

Reply:
I'd say everything should be formatted to 80-100 columns or so :)

Having paragraphs all on one line makes it difficult to edit especially in line-based editors like Vim.

That being said, I realise I'm the only one using Vim so we should go with whatever the majority wants.

Reply:
If all the other docs use long-lines, let's change this one.

Reply:
I don't think it's a big deal either way, we can merge it